package.json

```diff
@@ -65,6 +66,7 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+    "@iconify/react": "^5.0.1",
```

Comment:
The design from Alessio uses multiple iconsets, so `iconify/react` allows us to use multiple icon sets at once.